tests/App/Entity/User.php

```diff
@@ -22,6 +22,10 @@
  * @ORM\Entity()
  * @ORM\Table("liip_user")
  */
+#[
```

Comment:
Can you change this, to the following, and see if this fix some errors? On Php 7.4 run.

#[ORM\Entity()]
#[ORM\Table(name: "liip_user")]